mllib/src/main/scala/org/apache/spark/ml/param/params.scala

```diff
@@ -905,6 +905,15 @@ trait Params extends Identifiable with Serializable {
   }
 }
 
+object Params {
```

Comment:
Shall we make this object package private (for cleaner docs)

Reply:
Yes. Done.